src/schema/query/fleet.ts

```diff
@@ -0,0 +1,25 @@
+type Args = {
+  id: string;
+  organizationId: string;
+};
+
+type Fleet = {
+  id: string;
+  organizationId: string;
+  name: string;
+};
+
+type Context = {
+  organization_id?: string;
+};
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default (
+  { id, organizationId }: Args,
+  { organization_id }: Context
```

Comment:
🚫 **[eslint]** <custom-rules/no-destructuring-context> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Destructuring of the second argument is prohibited in this function.
<!-- __reviewdog__:ChA1MjU2NDQwMzUyOWUwZDQzEgZlc2xpbnQ= -->